### ISSUE TITLE
CI build broken

### ISSUE THREAD

**Issue by SyrianSpock** [cvra/pid#18]
https://travis-ci.org/cvra/pid/builds/168674816

**Comment by xiongyumail** [comment]
```
language: cpp
compiler:
    - gcc

before_install:
    - sudo pip install pyaml jinja2
    - pushd ..
    - wget "https://github.com/cpputest/cpputest.github.io/blob/master/releases/cpputest-3.6.tar.gz?raw=true" -O cpputest.tar.gz
    - tar -xzf cpputest.tar.gz 
    - cd cpputest-3.6/
    - ./configure
    - make
    - sudo make install
    - popd

    - git clone https://github.com/cvra/packager ../packager
    - pushd .
    - cd ../packager
    - git checkout d15247
    - popd

before_script:
    - ../packager/packager.py
    - mkdir build/
    - cd build/
    - cmake ..

script: 
    - make
    - ./tests
```

**Comment by SyrianSpock** [comment]
Indeed, newer versions of `packager` should be installed via `pip`. The CI is fixed with 0491e5f7e752633655737af77742a5120046e76d and cad807aa74e0abf42aa6ee5d15db5fc73b7d7184
